test/integration_test/basic_test.go

```diff
@@ -100,6 +100,9 @@ var testStorageClusterBasicCases = []types.TestCase{
 			return cluster
 		},
 		TestFunc: InstallWithTelemetry,
+		ShouldSkip: func(tc *types.TestCase) bool {
+			return ci_utils.PxOperatorVersion.LessThanOrEqual(ci_utils.PxOperatorVer1_6)
```

Comment:
1.6.1 will also not have it.

Reply:
I planned to use 1.7 for checking that, but we may have different tags like 1.7.0-dev, 1.7.0-rc1. did some experiments seems `1.7-` should do the job,